### ISSUE TITLE
Admin credential

### ISSUE THREAD

**Issue by amirulahmad** [intrip/laravel-authentication-acl#145]
I have installed this extension on Laravel 5.3. After run command:
`php artisan authentication:install`
I got this:
`[InvalidArgumentException]            
  Invalid hasher [] chosen for Sentry.`

The admin credential doesn;t work as well: admin@admin.com:password.

**Comment by intrip** [comment]
Which version of php are you using?

**Comment by amirulahmad** [comment]
It's 5.6.10.

**Comment by intrip** [comment]
Can't reproduce the problem. What version of the package are you using?

**Comment by TeryaevNikita** [comment]
the same problem...

for reproducing the problem try to remove
/config/acl_sentry.php
/config/cartalyst.sentry.php

and then run 
php artisan authentication:install

**Comment by intrip** [comment]
You shouldn't remove the sentry configuration files otherwise sentry doesn't know which strategy you should use to save password

**Comment by itualoha** [comment]
I met the sample report, too.
After run command:
php artisan authentication:install
I got this:
[InvalidArgumentException] Invalid hasher [] chosen for Sentry.

My env:
1. php 5.6.19
2. apache httpd 2.4.18
3. laravel 5.3.30 
   laravel installation command: 
   composer create-project --prefer-dist laravel/laravel myfirstexample "5.3.*"
4.after i finished the installation of laravel, I removed the confliction of database/migrations.

other installation steps followed the installation documentation. How could I troubleshoot the problem?
